tests/test_server.py

```diff
@@ -202,8 +246,12 @@ async def test_version_removal(client, server):
         assert res.code == 200
         result = await client.get_project(id=project_id)
 
+        result = await client.get_setting(tid=env_id, id=data.AVAILABLE_VERSIONS_TO_KEEP)
```

Comment:
I would pass this in as a parameter to the method. 

Now, if this method always returns a 100, the test would still succeed. 
(i.e. the assertion depends on what you are testing and it can be easily avoided)